android/build.gradle

```diff
@@ -122,7 +129,7 @@ allprojects {
             project.version = "${json.version}-jitsi-${versionQualifierNumber}"
 
             task jitsiAndroidSourcesJar(type: Jar) {
-                classifier = 'sources'
+                archiveClassifier = 'sources'
```

Comment:
classifier deprecated starting gradle 8.0

Reply:
Let's drop the Gradle update, it's a major one and should be handled separately.